src/qq.jl

```diff
@@ -11,17 +11,34 @@ function qqbuild(x::AbstractVector, y::AbstractVector)
     return QQPair(qx, qy)
 end
 
+
+"""
+Generates a sequence of probability points of length `n`:
+
+``
+(k − a)/(n + 1 − 2a), k ∈ 1, ..., n
+``
+
+`a` should be ∈ [0,1]. See the references listed here:
+https://en.wikipedia.org/wiki/Q%E2%80%93Q_plot#Heuristics
+"""
+function ppoints(n, a=0.5)
+    start = (1-a)/(n + 1 - 2*a)
+    stop = (n-a)/(n + 1 - 2*a)
+    range(start, stop, length=n)
+end
+
 function qqbuild(x::AbstractVector, d::UnivariateDistribution)
     n = length(x)
-    grid = [(1 / (n - 1)):(1 / (n - 1)):(1.0 - (1 / (n - 1)));]
+    grid = ppoints(n)
```

Comment:
Is parameter `a` in `ppoints` really needed if the function is only called with one argument?

Reply:
Yes, I was not sure about this one.
Different software use different values for `a`:

- R: `a = n <= 10 ? 3/8 : 0.5`
- Matlab: `a = 0.5`
- Scipy: `a = 0.3175` (but the first and the last point are computed differently)
- Minitab: `a = 3/8`

Maybe somebody would like to use `ppoints` independently of `qqbuild` with more flexibility?  Or should `a` be even an argument of `qqbuild`?

The value hardly matter practically,  I do not have a preference here.

Reply:
I think I'd prefer to get rid of the function and just hardcode the range here (with a comment and references). We can generalize once it's needed and deemed useful but I think the generality is not needed right now.

Reply:
Ah, I missed that it is used in the function below as well. So I suggest just removing parameter `a` then.

Reply:
I agree, this leads to the smallest change. Will update it accordingly.